client/java/src/main/java/io/openlineage/client/transports/KafkaTransport.java

```diff
@@ -0,0 +1,35 @@
+package io.openlineage.client.transports;
+
+import io.openlineage.client.OpenLineage;
+import io.openlineage.client.Utils;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
```

Comment:
Readme mentions other Kafka options like `acks` or key/value serializers. Are they respected?

Reply:
Basically we only pass through properties for Kafka. Everything that Kafka client supports should be supported.